config-provisioning/src/main/resources/configdefinitions/config.provisioning.node-repository.def

```diff
@@ -1,9 +1,11 @@
 # Copyright 2017 Yahoo Holdings. Licensed under the terms of the Apache 2.0 license. See LICENSE in the project root.
 namespace=config.provisioning
 
-# Docker image to use in REST API responses. This must be a fully qualified name, including registry, but excluding
-# version. Example: my-docker-registry.domain.tld:8080/dist/vespa
-dockerImage string default="dummyImage"
+# The container registry to use, excluding the repository. Example: my-registry.example.com:9999
+containerRegistry string default=""
```

Comment:
Does it make sense to pass these as separate? I think we should just pass 1 string and let `DockerImage::fromString` figure it out